scripts/generate.jl

```diff
@@ -0,0 +1,41 @@
+using Clang
+
+# LIBXPRESS_HEADERS are those headers to be wrapped.
+const LIBXPRESS_INCLUDE = joinpath(ENV["XPRESSDIR"], "include") |> expanduser |> normpath
+const LIBXPRESS_HEADERS = [
+                           joinpath(LIBXPRESS_INCLUDE, header) for header in [
+                                                                               # "bindrv.h",
+                                                                               # "mmhttp.h",
+                                                                               # "mmquad.h",
+                                                                               # "mmsystem.h",
+                                                                               # "mmxprs.h",
+                                                                               # "mosel_dso.h",
+                                                                               # "mosel_mc.h",
+                                                                               # "mosel_ni.h",
+                                                                               # "mosel_nifct.h",
+                                                                               # "mosel_rt.h",
+                                                                               # "xprb.h",
+                                                                               # "xprb_cpp.h",
+                                                                               # "xprd.h",
+                                                                               # "xprm_mc.h",
+                                                                               # "xprm_ni.h",
+                                                                               # "xprm_rt.h",
+                                                                               # "xprnls.h",
+                                                                               "xprs.h",
+                                                                               # "xprs_mse_defaulthandler.h",
+                                                                               # "XPRSdefaultMipSolEnumHandler.java",
+                                                                               # "xslp.h",
+                                                                            ]
```

Comment:
do you have access do a xprl.h file?

Reply:
No, I don't have an `xprl.h` file. The above files are the only ones in my install directory.

Reply:
ok, I will look into it